libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/NativeLoginManager.kt

```diff
@@ -270,108 +266,194 @@ internal class NativeLoginManager(
         return requestBodyString.toRequestBody(mediaType)
     }
 
-    // region Headless, Password-Less Login Via One-Time-Passcode
+    // region Salesforce Identity API Headless Forgot Password Flow
 
-    override suspend fun submitOtpRequest(
```

Comment:
Github's diff engine misaligned this part of the diff.  The logic of `submitOtpRequest` is simpler further in the source.